packages/ui/v2/core/apps/AppCard.tsx

```diff
@@ -63,16 +63,21 @@ export default function AppCard({ app, credentials }: AppCardProps) {
               <InstallAppButton
                 type={app.type}
                 isProOnly={app.isProOnly}
+                wrapperClassName="[@media(max-width:260px)]:w-full"
                 render={({ useDefaultComponent, ...props }) => {
                   if (useDefaultComponent) {
                     props = {
                       onClick: () => {
-                        mutation.mutate({ type: app.type });
+                        mutation.mutate({ type: app.type, variant: app.variant, slug: app.slug });
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.